maven/templates/deploy.py

```diff
@@ -30,9 +30,23 @@
 import sys
 import tempfile
 
-print(os.getcwd())
+import zipfile
 
 
+# This ZipFile extends Python's ZipFile and fixes the lost permission issue
```

Comment:
@lolski we don't really have permissions issue while repacking JARs — there are no executables with `+x` bit

Reply:
Good point, addressed in a911aea04576669bfeae464158eb457613f1f20a